src/apps/convert.ts

```diff
@@ -2,9 +2,12 @@ import commandLineUsage from 'command-line-usage';
 import {Section} from 'command-line-usage';
 import minimist from 'minimist';
 import path from 'path';
+import {AnyAzureObject} from '../conversion/azure';
```

Comment:
Minor nit: I put a blank line between include blocks for node_modules, sibling folders, local folder.